packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -375,7 +375,12 @@ const RealAccountSignup = ({
             localStorage.removeItem('real_account_signup_wizard');
         }
 
-        if (modal_content[getActiveModalIndex()].action === 'signup' && e && !is_closing_create_real_account_modal) {
+        if (
+            modal_content[getActiveModalIndex()].action === 'signup' &&
+            e &&
+            e?.target.closest('.dc-icon') &&
+            !is_closing_create_real_account_modal
+        ) {
```

Comment:
@suisin can we have a better way to check if it is close icon instead of this 
```
 e && e?.target.closest('.dc-icon')
```

Reply:
@suisin-deriv Maybe we can write a separate reusable function to close the modal on clicking the Maybe later button and close icon